code-studio/src/js/components/progress/course_progress.jsx

```diff
@@ -3,26 +3,35 @@ import { connect } from 'react-redux';
 import { stageShape } from './types';
 import _ from 'lodash';
 import CourseProgressRow from './course_progress_row.jsx';
+import color from '../../color';
+
+const styles = {
+  flexHeader: {
+    padding: '5px 10px',
+    margin: '10px 0 0 0',
+    borderRadius: 5,
```

Comment:
@Bjvanminnen prefers not using units: https://github.com/code-dot-org/code-dot-org/pull/8756#discussion_r65787579